Detectors/MUON/MCH/Simulation/src/DEDigitizer.cxx

```diff
@@ -78,6 +84,24 @@ void DEDigitizer::processHit(const Hit& hit, const InteractionRecord& collisionT
   auto localX = mResponse.getAnod(lpos.X());
   auto localY = lpos.Y();
 
+  // calculate angle between track and wire assuming wire perpendicular to z-axis
+  auto thetawire = asin((lexit.Y() - lentrance.Y()) / hitlengthZ); // check sign convention between O2 and Aliroot
+
+  // local b-field
+  double b[3] = {0., 0., 0.};
+  double x[3] = {lentrance.X(), lentrance.Y(), lentrance.Z()};
```

Comment:
the field is retrieved from the track position in the global coordinate system.

Reply:
Yes, it should be using the global coordinates.